db/seeds.rb

```diff
@@ -1,4 +1,6 @@
-ActiveFedora.fedora.connection.send(:init_base_path)
+if ENV["FCREPO_HOST"].present?
```

Comment:
we talked about moving this conditional to the application seeds.

avoiding putting it here will help these engine seeds support a wider variety of applications (i.e. those that don't configure with `FCREPO_HOST` should still expect `:init_base_path` to succeed)